runtime/cere/src/lib.rs

```diff
@@ -1399,7 +1394,7 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
-	InitiateNominationPools,
+	(InitiateNominationPools, pallet_nomination_pools::migration::v3::MigrateToV3<Runtime>),
 >;
```

Comment:
Do we need this line only for the `cere` runtime? I don't see such line in `cere-dev`